aio/deploy/helm-chart/kubernetes-dashboard/templates/ingress.yaml

```diff
@@ -79,6 +84,6 @@ spec:
   {{- end -}}
   {{- if .Values.ingress.tls }}
   tls:
-{{ toYaml .Values.ingress.tls | indent 4 }}
+{{ toYaml .Values.ingress.tls | nindent 4 }}
```

Comment:
hehe this could have been indented as well, but honestly thanks for the work so far in streamlining good usage of newlines.

Reply:
with pleasure :)